System/Posix/Terminal/ByteString.hsc

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE CApiFFI #-}
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
Actually, this pragma is not needed here, the synonyms are bundled in the defining module...  I'll revert this after CI passes, or if a fixup is needed to get it to pass.

Reply:
Done.